src/player.js

```diff
@@ -0,0 +1,26 @@
+export class Player{
+  constructor(newID){//finish reading on WeakMaps for private props
+    this.playerID = newID;
+    this.postition = null;
+    this.money = null;
+    this.propertiesOwned = new Array(28);
+    this.cardsOwned = null; //change to list or something;
+    this.isJailed = false;
+  }
+
+  static transferMoney(FromPlayer, ToPlayer, amount){
+    if(amount > FromPlayer){
+      //throw error, inssuficient funds;
+    }
+
+    FromPlayer.setMoney(FromPlayer.getMoney() - amount);//take money
+    ToPlayer.setMoney(ToPlayer.getMoney() + amount);//give money
+  }
+
+  set setMoney(amount){//ask Jake about set and get
```

Comment:
I can explain get and set, but for these, they're honestly not needed here.  

So typically in javascript you shouldn't have get or set functions.  Java/C++ require those because they don't support properties (the get/set keywords that I'll explain). 

So let's say we have a class that looks like this:

```
class Circle {
    constructor(radius) {
        this.radius = radius;
        this.area = Math.PI * radius * radius;
        this.circumference = 2 * Math.PI * radius;       
    }
}

const c = new Circle(10);
assert(c.radius == 10);
assert(c.area == 314.16);
assert(c.circumference == 6.2832);  // or whatever

// The above asserts should pass, but what if we want to change the radius:
c.radius = 100;
// Well the radius would update, but the area/circumference would still be the same. 
// We COULD turn area and circumference into functions, but we would end up losing 
//     the niceness that you get when they're properties.   Also extra functions pollute the code base.
//     This is what you do in Java because functions are the ONLY way to solve the problem.  But not so in 
//     Javascript

class Circle {
     constructor(radius) {
         this.radius = radius;
     }

     get area() {
         return this.radius * this.radius * Math.PI;
     }
     
     set area(newArea) {
          this.radius = Math.sqrt(newArea/Math.PI);
     }

     get circumference() {
          return this.radius * 2 * Math.PI;
     }

    set circumference(c) {
         this.radius = c/ (2 * Math.PI);
    }
}

// With this second version you can still access area/circumference like they were normal properties, but when you update them, radius will automatically update.   
```